tests/tool/Integration.Tests/E2ETest.cs

```diff
@@ -26,10 +26,12 @@ public class E2ETest
             ".upgrade-assistant"
         };
 
-        [InlineData("AspNetMvcTemplate", "TemplateMvc.csproj", "")]
-        [InlineData("WpfSample", "BeanTrader.sln", "BeanTraderClient.csproj")]
+        [InlineData("AspNetMvcTemplate.", "csharp", "TemplateMvc.csproj", "")]
+        [InlineData("WpfSample", "csharp", "BeanTrader.sln", "BeanTraderClient.csproj")]
+        [InlineData("AspNetMvcTemplate", "vb", "SinglePageApp.vbproj", "SinglePageApp.csproj")]
```

Comment:
"SinglePageApp.csproj" doesn't seem like the right entrypoint for SinglePageApp.vbproj. That can probably be empty instead.